HW02_B_Mimic_starter.Rmd

```diff
@@ -1,99 +1,118 @@
+
+
+### Graph 1
+```{r, echo = T, include = TRUE}
+
+ggplot(data = diamonds) + 
+  geom_bar(mapping = aes(x = cut, fill = clarity), position = "dodge") +
+  theme(plot.title = element_text(hjust = 0.5)) +
+  labs(x = "Diamond Cut", 
+       y = "Number of Diamonds", 
+       title = "My Diamond Collection", 
+       subtitle = "Barplot representing the number of diamonds in my diamond collection by \ntype of cut quality and clarity of diamond") +
+  annotate("rect", xmin = 4.5, xmax = 5.5, ymin = 0, ymax = 5000,
+           fill = "gray30", alpha  = 0.5,
+           label = "My Best Diamonds,\nof Course") +
+  annotate("text", x = 4, y = 4500, label = "My Best Diamonds,\nof Course") 
+
+```
+
+### Graph 2
+```{r, echo = T, include = TRUE}
+ggplot(data = iris, mapping = aes(x = Sepal.Length, y = Petal.Length)) + 
+  geom_point(mapping = aes(color = Species, shape = Species)) +
+  facet_wrap(~Species, scales = "free_y") +
+  geom_smooth(method = "lm", se = FALSE, color = "black")
+  labs(x = "Sepal.Length", 
+       y = "Petal.Length") 
+```
+
+
+### Graph 3
+You'll need the information in this first box to create the graph
+```{r, echo = TRUE}
+data("mpg")
+corvette <- mpg[mpg$model == "corvette",]
+#install
+require("ggrepel") #useful for making text annotations better, hint hint
+set.seed(42)
+```
+
+Now using the mpg dataset and the corvette dataset, make this graph:
+
+```{r graoh 3 code}
+ggplot() + 
+  geom_point(data = mpg, mapping = aes(x = displ, y = hwy)) +
+  scale_x_continuous(limits = c(1, 8), breaks = seq(1, 9, 1)) +
+  ggtitle("Corvettes are a bit of an outlier") +
+  geom_point(data = corvette, mapping = aes(x = displ, y = hwy), color = "blue") +
+  geom_text_repel(data = corvette, 
+                  mapping = aes(x = displ, y = hwy, 
+                  label = paste("Corvette", corvette$year, sep = ", ")), nudge_x = .4, nudge_y = .3)
+```
+
+There is a trick to getting the model and year to print off together. `paste()` is a useful function for this, also pasting together parts of file names and parts of urls together. 
+
+
+### Graph 4 
+```{r, echo = T}
+data(mpg)
+
+#hint for the coloring, colorbrewer and you can set palette colors and make your graphs colorblind friendly
+library(RColorBrewer)
+display.brewer.all(colorblindFriendly = T) #take a look at the colorblindfriendly options
+```
+
+The above graph lets you see some colobrlind friendly palettes. For the graph below, I used Set2.
+
+
+
+Now using the above mpg dataset, make this graph
+
+```{r graph 4 code}
+ggplot(data = mpg, mapping = aes(x = class, y = cty)) + 
+  scale_fill_brewer(palette="Set2") +
+  geom_point(mapping = aes(color = class), position = position_jitter(height = 0)) +
+  scale_color_brewer(palette = "Set2") +
+  geom_boxplot(outlier.alpha = 0, alpha = 0) +
+  coord_flip() +
+  labs(title = "Horizontal BoxPlot of City MPG and Car Class",
+       y = "City MPG",
+       xq = "Car Class")
```

Comment:
I think here is a typo xq = "Car Class", so that the x-axis title did not change in the image.